Tester.py

```diff
@@ -0,0 +1,30 @@
+from src.pykeen.pipeline import pipeline
```

Comment:
you should be installing pykeen using `pip install -e .` for development mode then just doing normal imports

Reply:
Thanks for this. Done.